app/views/payment/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="login_container">
+  <%= form_with model: @payment, url: group_payment_index_path, local: true do |f| %>
+    <div class="nav_bar">
+      <%= link_to group_index_path, class: "return_button" do %>
+        <%= image_tag('back.svg', class: "return_arrow") %>
+      <% end %>
+      <span class="nav_title">NEW TRANSACTION</span>
+      <button class="right_side_button">ADD </button>
+    </div>
+    <section class="login_section">
+      <div class="form_control">
+        <label for="trans_name" class="input_label">Name : </label>
+        <%= f.text_field :name, autofocus: true, placeholder: "name", class:"form_text_input input trans_input"  %>
+      </div>
+      <div class="form_control">
+        <label for="trans_amount" class="input_label">Amount : </label>
+        <%= f.number_field :amount, autofocus: true, placeholder: "0", class:"form_text_input input  trans_input"  %>
+      <!-- </div>
+      <div class="form_control">
+        <label for="trans_category" class="input_label">Category : </label>
+        <%= f.select :group_id, @user.groups.map { |group| [group.name, group.id]} ,class: "select_input" %>
+      </div> -->
+      
+    </section>
+  <%end%>
```

Comment:
- [ ] It will be great to ensure that in your form for creating a new transaction, there is a way to select a category. At least one category. This is a major requirement for this project. One of the issues that is causing a problem here is that you have not set up your project associations correctly. I have added a comment about the associations on the review, kindly read that carefully. ⬇️  
![Screenshot 2023-06-11 at 07 55 55](https://github.com/hopeforkoami/Budget_App/assets/32361963/9fa26faf-8977-4aac-9e54-bb506bebc71c)

Reply:
- [ ] I have not been able to check if this change has been implemented well since there is an error popping up when I try to create a group. Kinly fix that error and then the next reviewer will be able to check for this requirement.